pkg/controller/controller.go

```diff
@@ -155,9 +154,8 @@ func (c *controller) Run(workers int, stopCh <-chan struct{}) {
 	for i := 0; i < workers; i++ {
```

Comment:
@MHBauer There shouldn't be. The queue type is thread-safe, and ensures that no two threads can be operating on one key simultaneously. So we should just have a bunch of threads working on distinct instances.